Automattic/wp-calypso
# Update/fix coupons ui on mobile

Currently in cart there are two links: "Have a coupon code?" and "Show order summary". None of them works. Order summary won't show and coupon code form won't work. This PR fixes those problems.

**Test plan**

1. Go to checkout for any product either in responsive view or on a mobile device
1. Scroll to the bottom of the page
1. Click "Show order summary"
1. An order summary should appear and the link should change to "Hide order summary"
1. Click it again and confirm the order summary disappeared
1. Click on "Have a coupon code?" link
1. Apply a coupon code, confirm it worked
